### ISSUE TITLE
Yarn audit reports issue with protobuf dependency

### ISSUE THREAD

**Issue by mschilde** [firebase/firebase-admin-node#1868]
Yarn audit reports the following:

```
│ high          │ Prototype Pollution in protobufjs                            │
───────────────────
│ Package       │ protobufjs                                                   │
───────────────────
│ Patched in    │ >=6.11.3                                                     │
───────────────────
│ Dependency of │ [...]                                     │
───────────────────
│ Path          │ [...] >        │
│               │ firebase-admin > @google-cloud/firestore > google-gax >      │
│               │ @grpc/grpc-js > @grpc/proto-loader > protobufjs              │
───────────────────
│ More info     │ https://www.npmjs.com/advisories/1075704  
```

It looks like Firestore fixed the issue in the meantime, see https://github.com/googleapis/nodejs-firestore/issues/1740

What's the timeline for fixing the security issue on `firebase-admin`?

**Comment by lahirumaramba** [comment]
This is fixed in the latest version of `firebase-admin` (I believe the fix was in `v10.3.0`). Could you confirm your Admin SDK version, please?

**Comment by mschilde** [comment]
You're right, I'm very sorry for the false alarm. The issue was fixed after regenerating the `yarn.lock` file, which was patched previously by the dependabot.

Keep up the good work 👍
